src/components/CommonComponents/SearchBar/index.tsx

```diff
@@ -91,15 +91,17 @@ const SearchBar = (): JSX.Element => {
     }
   };
 
+  const toggleContainer = () => isExpanded ? collapseContainer() : expandContainer();
```

Comment:
Prettier is failing in CI because this line is too long. `npm run format` will fix